docs/source/develop/index.rst

```diff
@@ -6,7 +6,7 @@ Iroha is a great backend for your applications working on distributed ledgers.
 In this section we will cover the instruments (API reference and libraries) and cases you could use for that.
 
 You can also check out `The Borsello App <https://github.com/claudiocandio/borsello>`_ – it is a wallet for Android & iOS along with a web browser application contributed by `Claudio <https://github.com/claudiocandio>`_.
-It might help you understand the general idea of how frontend can be used with HL Iroha.
+It might help you understand the general idea of how frontend can be used with HL Iroha. You can also check out `sample web javascript wallet <https://github.com/soramitsu/iroha-wallet-js>`_ developed in `Vue <https://vuejs.org/>`_.
```

Comment:
```suggestion
It might help you understand the general idea of how frontend can be used with HL Iroha. You can also check out `the sample web javascript wallet <https://github.com/soramitsu/iroha-wallet-js>`_ developed in `Vue <https://vuejs.org/>`_.
```